package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vsoagent-installer",
   "description": "Visual Studio Xplat Build Agent Installer",
   "main": "bin/install.js",
-  "version": "0.2.28",
+  "version": "0.2.29",
```

Comment:
This needs to be 0.2.30.  The reason you don't see 0.2.29 is I didn't push this file back.  But npm currently has 0.2.29.  We need 0.2.30 for it to accept an update.